src/core/search/indices.cc

```diff
@@ -75,4 +89,13 @@ void TagIndex::Add(DocId doc, string_view value) {
   }
 }
 
+void TagIndex::Remove(DocId doc, string_view value) {
+  auto tags = absl::StrSplit(value, ',');
+  for (string_view tag : tags) {
+    auto& list = entries_[absl::StripAsciiWhitespace(tag)];
```

Comment:
Maybe a stupid question: here and in `Add()`, don't we need to lowercase `tag`?

Reply:
Indeed, we need to